frontend/src/components/workspaces/Templates/TemplatesTable/TemplatesTable.tsx

```diff
@@ -0,0 +1,146 @@
+      id: number;
+      name: string;
+      ip: string;
+      status: boolean;
+    }>;
+  }>;
+  role: WorkspaceRole;
+  editTemplate: (id: string) => void;
+  deleteTemplate: (id: string) => void;
+}
+
+const TemplatesTable: FC<ITemplatesTableProps> = ({ ...props }) => {
+  const { templates, role, editTemplate, deleteTemplate } = props;
+
+  /**
+   * Our Table has just one column which render all rows using a component TemplateTableRow
+   */
+
+  const columns = [
+    {
+      title: 'Template',
+      key: 'template',
+      /* eslint-disable react/no-multi-comp */
+      render: (record: {
+        id: string;
+        name: string;
+        gui: boolean;
+        instances: Array<{
+          id: number;
+          name: string;
+          ip: string;
+          status: boolean;
+        }>;
+      }) => (
+        <TemplatesTableRow
+          id={record.id}
+          name={record.name}
+          gui={record.gui}
+          role={role}
+          activeInstances={record.instances ? record.instances.length : 0}
+          editTemplate={editTemplate}
+          deleteTemplate={deleteTemplate}
+        />
+      ),
+      /* eslint-enable react/no-multi-comp */
+    },
+  ];
+
+  /**
+   * Handle to manage accordion effect, it's possible to allow multiple row expansion (no accordion effect) by switching
+   * handleAccordion() function with the following code
+    const handleAccordion = (
+      expanded: boolean,
+      record: {
+        id: string;
+        name: string;
+        gui: boolean;
+        instances: Array<{
+          id: number;
+          name: string;
+          ip: string;
+          status: boolean;
+        }>;
+      }
+    ) => {
+      expanded
+        ? setExpanedId(expandedId => {
+            expandedId.push(record.id);
+            return expandedId;
+          })
+        : setExpanedId(expandedId => {
+            return expandedId.filter(id => id !== record.id);
+          });
+    };
+   */
+  const [expandedId, setExpandedId] = useState(['']);
+
+  const handleAccordion = (
+    expanded: boolean,
+    record: {
+      id: string;
+      name: string;
+      gui: boolean;
+      instances: Array<{
+        id: number;
+        name: string;
+        ip: string;
+        status: boolean;
+      }>;
+    }
+  ) => {
+    expanded ? setExpandedId([record.id]) : setExpandedId(['']);
+  };
+
+  return (
+    <div
+      className="w-full flex-grow flex-wrap content-between py-0 overflow-auto scrollbar"
+      style={{ height: 'calc(100vh - 380px)', minHeight: '171px' }} //465px if we want to add a Box footer
+    >
+      <Col span={24} className="flex-auto">
+        <Table
+          size={'small'}
+          showHeader={false}
+          rowKey={record => record.id}
+          columns={columns}
+          dataSource={templates}
+          pagination={false}
+          onExpand={handleAccordion}
+          expandable={{
+            rowExpandable: record => (record.instances.length ? true : false),
```

Comment:
```suggestion
            rowExpandable: record => (!!record.instances.length),
```
This could be a fast way to "cast" stuff to boolean, if @GabriFila approves 🥴

Reply:
Changed it